lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -673,5 +673,6 @@ def _section_metrics(course, access):
         'get_students_problem_grades_url': reverse('get_students_problem_grades'),
         'post_metrics_data_csv_url': reverse('post_metrics_data_csv'),
         'enrollment': CourseEnrollment.num_enrolled_in(course_key),
+        'lms_base': settings.LMS_BASE,
```

Comment:
On second thought, do we really even need this variable?
Since all we're doing is linking to the same domain that we're already on,
can't we just omit the domain entirely, similar to how you're already implying the protocol?

```javascript
url = '/courses/whatever_else_we_need';
```